clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -2652,6 +2652,31 @@ void Generic_GCC::GCCInstallationDetector::AddDefaultGCCPrefixes(
     return;
   }
 
+  if (TargetTriple.isOSHurd()) {
+    static const char *const X86_64HurdTriples[] = {"x86_64-gnu"};
```

Comment:
Since there is just one, `TripleAliases.push_back("x86_64-gnu")` will be clearer and less verbose.

Reply:
But then also `BiarchTripleAliases.push_back("x86_64-gnu");` ?

Reply:
No. It's for `-m32`/`-m64`